microsoft-edge/devtools-guide-chromium/accessibility/preferred-color-scheme-simulation.md

```diff
@@ -10,35 +10,34 @@ keywords: microsoft edge, web development, f12 tools, devtools
 ---
 # Dark or light color scheme simulation  
 
-Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system is grating and can be an accessibility issue for some users.  
+Many operating systems have a way to display any application in darker or lighter colors.  Having a web product that has a light theme in a dark-mode operating system can be hard to read and can be an accessibility issue for some users.  
 
 For a webpage, you can use the [prefers-color-scheme][MDNPrefersColorScheme] CSS Media Query to detect whether the user prefers to display your product in a darker or lighter color scheme.  Then to test the result, use [Microsoft Edge DevTools][DevtoolsIndex] to simulate a change from dark to light mode, without having to change the setting for your entire operating system.  
 
 **To emulate dark or light theme:**
 
-1.  Open the **Command Menu**.  
-    1.  Select `Ctrl`+`Shift`+`P` \(Windows/Linux\) or `Command`+`Shift`+`P` \(macOS\).  
-        
-        :::image type="complex" source="../media/css-console-command-menu-rendering.msft.png" alt-text="The Command Menu" lightbox="../media/css-console-command-menu-rendering.msft.png":::
-           The **Command Menu**  
-        :::image-end:::  
-        
-1.  Type `emulate color`, choose either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light** and then select `Enter`.  
+1.  Open the **Command Menu** by selecting `Ctrl`+`Shift`+`P` \(Windows/Linux\) or `Command`+`Shift`+`P` \(macOS\).  
+
+    :::image type="complex" source="../media/css-console-command-menu-rendering.msft.png" alt-text="The Command Menu" lightbox="../media/css-console-command-menu-rendering.msft.png":::
+        The **Command Menu**  
+    :::image-end:::  
+
+1.  Type "emulate color", choose either **Emulate CSS prefers-color-scheme: dark** or **Emulate CSS prefers-color-scheme: light**, and then select **Enter**.  
```

Comment:
```suggestion
1.  Type "dark" or "light", select **Rendering: Emulate CSS prefers-color-scheme: dark** or **Rendering: Emulate CSS prefers-color-scheme: light**, and then select **Enter**.  
```
Simply typing "dark" or "light" is more realistic and advisable than the harder-to-type & harder-to-remember phrase, "emulate color".  But changing this text requires redoing the screen capture too.